README.md

```diff
@@ -71,21 +71,21 @@ The event and query functionality of the CLI supports a number of different ways
 ####Recent Events
 
 The 'get recentevents' command allows you to retrieve the most recent log events that have been sent to Logentries.
-The logs to retrieve events from can be specified in a few ways. The Log IDs can be passed directly as a space separated list of log Ids, or you can take advantage of log groups and log nicknames. Log Ids can be obtained from the settings page or log set page of a log in the Logentries UI (https://logentries.com). Log nicknames can be passed using the '--lognick' '-n' arguments, log groups can be passed using the '--loggroup' '-g' arguments. For more information in setting up log nicknames and log groups, see the 'Log Nicknames and Groups' section below.
+The logs to retrieve events from can be specified in a few ways. The Log IDs can be passed directly as a space separated list of log Ids, or you can take advantage of log sets and CLI Favorites. Log Ids can be obtained from the settings page or logsets page of a log in the Logentries UI (https://logentries.com). CLI favorites can be passed using the '--favorites' '-c' arguments, logset ID can be passed using the '--logset' '-g' arguments. For more information in setting up CLI Favorites and using log sets, see the 'CLI Favorites and Log Sets' section below.
 By default the 'get recentevents' command will return events for the last 20 minutes. The command also takes an optional time argument that allows you to specify how far back in time you wish to get events from; this is passed using '--last' or '-l' argument.
 It is also possible to provide '-r' (--relative-range) to use relative time range functionality of the Logentries REST API. Check [supported patterns](#supported-relative-time-patterns).
 
 Example usage: 
 ```
 lecli get recentevents 12345678-aaaa-bbbb-1234-1234cb123456 -l 200
 lecli get recentevents 12345678-aaaa-bbbb-1234-1234cb123456 -r 'last 2 hours'
-lecli get recentevents -n mynicknamedlog -l 200
-lecli get recentevents -g myloggroup -l 200
-lecli get recentevents -g myloggroup -r 'last 1 week'
+lecli get recentevents -c mylogalias -l 200
+lecli get recentevents --logset 12345678-aaaa-bbbb-1234-1234cb123457 -l 200
```

Comment:
Would it be possible to support logset by name? Seems very unintuitive to have to get the uuid.

Reply:
I discuss this with Eoin and we agreed to do it this way, based on the following....

In the existing logset management api on lecli the user already uses logset ID for deletion, renaming and to get a specific logset; also it is more safer to use an ID as logset names on the web app are not  case insensitive meaning the wrong one could be used; also for implementation this way requires less requests and can use existing code.